backend/tests/apps/mentorship/api/internal/nodes/test_api_internal_mentee.py

```
@@ -0,0 +1,61 @@
+    assert mock_mentee_node.resolve_experience_level() == ExperienceLevelEnum.BEGINNER
+
+
+def test_mentee_node_resolve_experience_level_none():
+    """Test the resolve_experience_level method when experience_level is None."""
+    mentee_node_no_exp = MenteeNode(
+        id="2",
+        login="noexpmentee",
+        name="No Experience Mentee",
+        avatar_url="https://example.com/noexp.jpg",
+        bio=None,
+        experience_level=None,  # type: ignore
```

Ruff (0.14.10)

Use specific rule codes when ignoring type issues

(PGH003)